README.md

```diff
@@ -3,17 +3,74 @@
 
 Telegram bot - [@lilmonix3Bot](https://t.me/lilmonix3Bot). A multipurpose bot.
 
+**Table of Content**
+
+- [Current features](#current-features)
+	- [Urban Dictionary](#urban-dictionary)
+	- [Decide](#decide)
+	- [Roll dice](#roll-dice)
+	- [Table](#table)
+	- [Shrug](#shrug)
+	- [Credits](#credits)
+	- [Quote](#quote)
+	- [Regex](#regex)
+- [Donate](#donate)
+- [Want more features?](#want-more-features?)
+
 ## Current features
 These features are currently implemented -
-* Urban Dictionary (/urban_dictionary <TERM>)
-* Decide (/decide)
-* Dice roll (/roll_dice)
-* Table flip/unflip (/table flip|unflip)
-* Shrug (/shrug)
-* Credits (/credits)
-* Quotes (/quotes)
-* Regex (This is not a command. It is like s/regex text/replaced text. To use this, reply to a message with this syntax. TODO: Add example here.)
-TODO: Add examples.
+
+### Urban Dictionary
+#### Purpose:
+Give the meaning of a word from [Urban Dictionary](www.urbandictionary.com/).
+#### Usage:
+`/urban_dictionary [WORD]`
+
+### Decide
+#### Purpose:
+Helping people in dilemma with 45% chances of the bot replying `Yes` (to do something, buy a new phone, relase your North Korean missile, etc.), 45% chances of replying `No`(no to downloading Vim, sing in shower, die because of not doing homework, etc.), and 10% chances of replying `Maybe`(putting people in dilemma again).
+#### Usage:
+`/decide`
+
+### Roll dice
+#### Purpose:
+Helping people play Ludo (who can't buy a physical kit or don't have the intellect to download an app for it) by outputing a random number between 1 and 6.
+#### Usage:
+`/roll_dice`
+
+### Table
+#### Purpose:
+Helping people have know that you freaked out because they're not capitalizing appropriate letters or that the finally capitalized it.
+#### Usage:
+`/table flip`: (╯°□°）╯彡 ┻━┻
+
+`/table unflip`: ┬─┬ ノ(°-°ノ)
+
+### Shrug
+#### Purpose:
+Helping people express that they're a human and not a bot reply "**Idk**" all the time.
+#### Usage:
+`/shrug`: ¯\_(ツ)_/¯
+
+
+### Credits
+#### Purpose:
+Popularize some non-famous coders.
+#### Usage:
+`/credits`
+
+### Quote
+#### Purpose:
+Markdown some of the messages that you'll be needing later.
+#### Usage:
+`[Reply to someone] /quote`: Append the message you replied to the list of quotes.
+
+`/quote [index]`: Show the quote on the index you typed.
+
+`/quote all`: List all the quotes ever.
+
+### Regex
+To be added later.
```

Comment:
Usage is like this -
Someone sent `String 1: Hello this is a string.`
You reply to String 1, with text - `s/Hello/Bye`.
The bot will reply `String 2: Bye this is a string.`
It uses the Python regex format, and is intended to be used like `sed`.

Purpose is to replace text with the bot.